migrations/2_deploy_contracts.js

```diff
@@ -161,6 +162,23 @@ module.exports = async function (deployer, network) {
     }
   }
 
+  for(const fee of currentNetworkConfig.fee) {
```

Comment:
Will this work for the ERc1155 and generic which are not in this migration?
Might be better to add this migration as separate at the end.

@nmlinaric  WDYT?

Reply:
It will work because we actually don't need to deploy the resource to register the route. For registration we just need the destinationDomainID, resourceID and address of the feeHandler. But I can move it into separate migration file at the end of all resource deployments, since it is more intuitive that way